cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -143,7 +148,8 @@ public class KafkaReconciler {
     private final AdminClientProvider adminClientProvider;
 
     private final int currentReplicas;
-
+    private static int replicasBeforeScaleDown;
+    private static boolean canScaleDown;
```

Comment:
Why should these be static? This has to cause conflicts when reconciling multiple clusters in parallel.